src/drivers/github.go

```diff
@@ -8,21 +8,46 @@ import (
 
 	"golang.org/x/oauth2"
 
-	"github.com/git-town/git-town/src/git"
 	"github.com/google/go-github/github"
 )
 
+type githubConfig interface {
+	GetCodeHostingDriverName() string
+	GetMainBranch() string
+	GetURLRepositoryName(string) string
+	GetURLHostname(string) string
+	GetCodeHostingOriginHostname() string
+	GetRemoteOriginURL() string
+	GetGitHubToken() string
+}
+
 type githubCodeHostingDriver struct {
 	originURL  string
 	hostname   string
 	apiToken   string
 	client     *github.Client
 	owner      string
 	repository string
+	mainBranch string
```

Comment:
Wouldn't that be actually `parentBranch` given the usage? :space_invader:

Reply:
I think you are right.